lib/parsers.js

```diff
@@ -38,6 +40,28 @@ exports.parseMessage = function (msg) {
   return error
 }
 
+exports.parseStackTrace = function (err) {
+  err = err.originalError || err
```

Comment:
I'd prefer a comment saying which framework we normally expect `originalError` to come from as this isn't a standard thing